codegen/CMakeLists.txt

```diff
@@ -50,6 +50,4 @@ rocm_install(
 )
 rocm_install(DIRECTORY include/ck DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
-if(BUILD_TESTING)
 add_subdirectory(test)
-endif()
```

Comment:
Why are we removing this? The tests should be conditionally built. I dont think downstream users wants to waste time compiling extra tests(its really only needed for developers).